parsec/lib/parsec.dart

```diff
@@ -0,0 +1,14 @@
+// You have generated a new plugin project without specifying the `--platforms`
+// flag. A plugin project with no platform support was generated. To add a
+// platform, run `flutter create -t plugin --platforms <platforms> .` under the
+// same directory. You can also find a detailed instruction on how to add
+// platforms in the `pubspec.yaml` at
+// https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms.
+
+import 'package:parsec_platform_interface/parsec_platform_interface.dart';
+
+class Parsec {
+  Future<dynamic> eval(String equation) {
+    return ParsecPlatform.instance.eval(equation);
```

Comment:
the internal method should continue be called nativeEval in order to highlight to us, that it calls native code. but the external method that the plugin users will use should be called just eval for simplicity.
```suggestion
    return ParsecPlatform.instance.nativeEval(equation);
```